core-bundle/src/Resources/contao/templates/modules/mod_breadcrumb.html5

```diff
@@ -1,15 +1,14 @@
-<?php $this->extend('block_unsearchable'); ?>
 
-<?php $this->block('content'); ?>
-
-  <ul itemprop="breadcrumb" itemscope itemtype="http://schema.org/BreadcrumbList">
-    <?php foreach ($this->items as $position => $item): ?>
-      <?php if ($item['isActive']): ?>
-        <li class="active<?php if ($item['class']): ?> <?= $item['class'] ?><?php endif; ?> last"><?= $item['link'] ?></li>
-      <?php else: ?>
-        <li<?php if ($item['class']): ?> class="<?= $item['class'] ?>"<?php endif; ?> itemscope itemtype="http://schema.org/ListItem" itemprop="itemListElement"><a href="<?= $item['href'] ?: './' ?>" title="<?= $item['title'] ?>" itemprop="item"><span itemprop="name"><?= $item['link'] ?></span></a><meta itemprop="position" content="<?= $position + 1 ?>"></li>
-      <?php endif; ?>
-    <?php endforeach; ?>
-  </ul>
-
-<?php $this->endblock(); ?>
+<!-- indexer::stop -->
+<nav class="<?= $this->class ?> block"<?= $this->cssID ?><?php if ($this->style): ?> style="<?= $this->style ?>"<?php endif; ?> aria-label="Breadcrumb" itemscope itemtype="http://schema.org/BreadcrumbList">
+    <ul>
+      <?php foreach ($this->items as $position => $item): ?>
+        <?php if ($item['isActive']): ?>
+          <li class="active<?php if ($item['class']): ?> <?= $item['class'] ?><?php endif; ?> last" aria-current="page"><?= $item['link'] ?></li>
+        <?php else: ?>
+          <li<?php if ($item['class']): ?> class="<?= $item['class'] ?>"<?php endif; ?> itemscope itemtype="http://schema.org/ListItem" itemprop="itemListElement"><a href="<?= $item['href'] ?: './' ?>" title="<?= $item['title'] ?>" itemprop="item"><span itemprop="name"><?= $item['link'] ?></span></a><meta itemprop="position" content="<?= $position + 1 ?>"></li>
+        <?php endif; ?>
+      <?php endforeach; ?>
+    </ul>
+  </nav>
+<!-- indexer::continue -->
```

Comment:
We also need the `headline` and `content` blocks in the template. Currently in Contao 4.9 you can do the following:

```php
<!-- templates/mod_breadcrumb.html5 -->
<?php $this->extend('mod_breadcrumb'); ?>

<?php $this->block('content'); ?>
  <p>Lorem ipsum</p>
  <?php $this->parent(); ?>
<?php $this->endblock(); ?>
```

I don't think we should break this in a bugfix release.

Reply:
Right, done